I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 295, in script
    call screen newspaper
ScriptError: could not find label 'chapter_III_newspaper'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\alext\OneDrive - Universidade de Lisboa\Documentos\Programas\renpy-8.0.0-sdk\renpy\bootstrap.py", line 274, in bootstrap
    renpy.main.main()
  File "C:\Users\alext\OneDrive - Universidade de Lisboa\Documentos\Programas\renpy-8.0.0-sdk\renpy\main.py", line 660, in main
    run(restart)
  File "C:\Users\alext\OneDrive - Universidade de Lisboa\Documentos\Programas\renpy-8.0.0-sdk\renpy\main.py", line 146, in run
    renpy.execution.run_context(True)
  File "C:\Users\alext\OneDrive - Universidade de Lisboa\Documentos\Programas\renpy-8.0.0-sdk\renpy\execution.py", line 938, in run_context
    context.run()
  File "game/script.rpy", line 295, in script
    call screen newspaper
  File "C:\Users\alext\OneDrive - Universidade de Lisboa\Documentos\Programas\renpy-8.0.0-sdk\renpy\script.py", line 927, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'chapter_III_newspaper'.

Windows-10-10.0.19044 AMD64
Ren'Py 8.0.0.22062402
SuNoFes 22 1.0
Fri Aug 19 22:58:39 2022
